aws-ecs-job-fargate/iam.tf

```
@@ -41,3 +41,26 @@ resource "aws_iam_role_policy" "task_execution_role_secretsmanager" {
   role   = aws_iam_role.task_execution_role.name
   policy = data.aws_iam_policy_document.registry_secretsmanager[0].json
 }
+
+data "aws_iam_policy_document" "ssm_permission" {
+  count = var.ssm_arn != null ? 1 : 0
+
+  statement {
+    actions = [
+      "ssm:GetParameters",
+    ]
+
+    resources = [var.ssm_arn]
```

```suggestion
    resources = var.ssm_parameter_store_arns
```
I'd make this a list so that we're granting access to specific keys in Parameter Store. Even though you can somewhat achieve the same behavior by providing an ARN prefix, I think this is much more flexible.